static/js/contact.js

```diff
@@ -68,7 +68,6 @@ form?.addEventListener('submit', async (event) => {
           contactAlert.style.display = 'none';
         }, 5000);
       }
-      return response.json();
```

Comment:
No data is returned from the BE if a form submission is successful and this caused an unexpected end of JSON input error to be caught in the `catch` block.